src/bodywork/k8s/workflow_jobs.py

```diff
@@ -97,6 +103,18 @@ def configure_workflow_job(
     return job
 
 
+def _create_job_name(project_repo_url: str, project_repo_branch: str) -> str:
```

Comment:
Do we have a test for this (implicit or explicit)?

Reply:
I remember have an implicit one somewhere but  I can't remember where, I had to mock it.

Reply:
test_configure_workflow_job